src/SessionSetup/RetrieveInstancesWidget.cpp

```diff
@@ -31,17 +31,13 @@ using InstanceListScope = orbit_ggp::Client::InstanceListScope;
 
 RetrieveInstancesWidget::~RetrieveInstancesWidget() = default;
 
-RetrieveInstancesWidget::RetrieveInstancesWidget(RetrieveInstances* retrieve_instances,
-                                                 QWidget* parent)
+RetrieveInstancesWidget::RetrieveInstancesWidget(QWidget* parent)
```

Comment:
Can you add in this first commit, a message explaining why this is needed. Difficult to get without context.

Reply:
Also commit message seem to miss "Test" (and "Bug") stanza.

Reply:
Yes, true. I forgot to add that to the commit message. I will change the message in a rebase to:

Make RetrieveInstancesWidget default constructible

This is needed so RetrieveInstancesWidget can be used in QDesigner files (.ui). For that the widget needs to have a constructor that takes only the parent as parameter and no other parameters

Reply:
Added a commit message in the last "address review commit" message. Added bugs and tests as well